### PR TITLE
Create Item model and CRUD actions

### PR DESCRIPTION
**Pull request by SashaShostyr**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -39,6 +39,7 @@ gem 'bootsnap', '>= 1.1.0', require: false
 
 gem 'bootstrap-sass', '~> 3.3', '>= 3.3.7'
 gem 'jquery-rails'
+gem 'bootstrap_form', '>= 4.0.0.alpha1'
```
Comment (Sergyenko) [inline Gemfile]: Why do we need this gem?

File: config/database.yml [hunk 2]
```diff
@@ -2,6 +2,8 @@ default: &default
   adapter: postgresql
   encoding: unicode
   pool: 5
+  username: nicher
+  password: <%= ENV['NICHER_DATABASE_PASSWORD'] %>
```
Comment (Sergyenko) [inline config/database.yml:6]: Why do you change it? Are you sure, that all team members have the same user?

File: config/database.yml [hunk 3]
```diff
@@ -14,5 +16,3 @@ test:
 production:
   <<: *default
   database: nicher_production
-  username: nicher
-  password: <%= ENV['NICHER_DATABASE_PASSWORD'] %>
```
Comment (Sergyenko) [inline config/database.yml:18]: Why you deleted it from default block?

**Review by Sergyenko — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (SashaShostyr) [reply Gemfile]: для создания формы в _form.html.erb

**Review by cybergizer — APPROVED** [review]

**Review by Sergyenko — APPROVED** [review]

**Pull request merged**